modules/utils/docker/bridge_settings.sh

```diff
@@ -50,20 +50,111 @@ if [ -f "$COMMS_PCB_VERSION_FILE" ]; then
 fi
 echo $eth_port
 
-# Set mtu back to 1500 to support e2e connectivity
-# TODO: Investigate this as it should still work with 1460
-ifconfig bat0 mtu 1500
-ifconfig br-lan mtu 1500
-
-brctl addif br-lan bat0 $eth_port
-echo $br_lan_ip
-ifconfig br-lan $br_lan_ip netmask "255.255.255.0"
-ifconfig br-lan up
-echo
-ifconfig br-lan
+create_dhcpd_config()
+{
+  SUBNET="$1"
+
+  cat > /etc/dhcp/dhcpd.conf <<- EOF
+  default-lease-time 600;
+  max-lease-time 7200;
+  ddns-update-style none;
+  authoritative;
+
+  subnet $SUBNET.0 netmask 255.255.255.0 {
+    range $SUBNET.100 $SUBNET.199;
```

Comment:
Should be `range $SUBNET.2 $SUBNET.254;` 

The .100 to .199 range sounds like a leftover from testing when other devices were already statically configured as .2, .3, etc :)